src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/ConnectionPluginManager.java

```diff
@@ -47,18 +49,22 @@
   protected PropertySet propertySet = null;
   protected IConnectionPlugin headPlugin = null;
   ClusterAwareConnectionProxy proxy;
-  private final Thread shutdownHook = new Thread(this::releaseResources);
+  protected static final Queue<ConnectionPluginManager> instances = new ConcurrentLinkedQueue<>();
```

Comment:
Nit: Static variables are usually defined at the top of class declaration, before other variables.

Reply:
Done.